### ISSUE TITLE
Get-process does not populate the Handles field. 

### ISSUE THREAD

**Issue by Francisco-Gamino** [PowerShell/PowerShell#2207]
## Steps to reproduce

Get-Process
## Expected behavior

The returned output contains the Handles fields
## Actual behavior

Handle property is always 0
PS D:> Get-Process

Handles  NPM(K)    PM(K)      WS(K)     CPU(s)     Id  SI ProcessName
---

```
  0      32    25108      26968       4.47   8844   1 ApplicationFrameHost
  0      13     7392      13196       0.11  22956   0 audiodg
```
## Environment data

``` powershell
> $PSVersionTable
Name                           Value
----                           -----
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSEdition                      Core
BuildVersion                   3.0.0.0
GitCommitId                    v6.0.0-alpha.9-246-g124535f68b1cc8faaa052631a7db97444abbbcee
PSRemotingProtocolVersion      2.3
CLRVersion
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
PSVersion                      6.0.0-alpha

```

**Comment by mirichmo** [comment]
System.Diagnostics.Process.HandleCount does not exist for CoreCLR. Have we filed a bug for them to add it back?

**Comment by iSazonov** [comment]
I not found such open CoreCLR Issue.
But code [Process.cs](https://github.com/PowerShell/PowerShell/blob/309bf603f9aff9009dce7e725d42b2d4f99f58c9/src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs) contain comments:
>// In CoreCLR, the System.Diagnostics.Process.HandleCount property does not exist. 
// I am adding a note property HandleCount and temporarily setting it to zero. 
// This issue will be fix for RTM and it is tracked by 5024994: Get-process does not populate the Handles field. 

There might be an internal MSFT process on this issue. If so, then it would be good to confirm it here that the bug will be fixed. (When?)

Related #1812

**Comment by SteveL-MSFT** [comment]
Closing this one, reopening #1812
